crates/holochain_conductor_api/src/app_interface.rs

```diff
@@ -2,109 +2,99 @@ use crate::{signal_subscription::SignalSubscription, ExternalApiWireError};
-    /// [`AppResponse::ZomeCall`]: enum.AppResponse.html#variant.ZomeCall
-    /// [`AppResponse::Error`]: enum.AppResponse.html#variant.Error
+    /// [`AppResponse::ZomeCall`]
     ZomeCall(Box<ZomeCall>),
 
-    /// DEPRECATED. Use `ZomeCall`.
+    #[deprecated = "use ZomeCall"]
     ZomeCallInvocation(Box<ZomeCall>),
 
-    /// Update signal subscriptions.
-    ///
     /// Is currently unimplemented and will return
-    /// an [`AppResponse::Unimplemented`](enum.AppResponse.html#variant.Unimplemented)
+    /// an [`AppResponse::Unimplemented`].
     SignalSubscription(SignalSubscription),
 }
 
-/// Responses to requests received on an App interface
+/// Represents the possible responses to an [`AppRequest`].
 #[derive(Clone, Debug, serde::Serialize, serde::Deserialize, SerializedBytes)]
 #[serde(rename_all = "snake_case", tag = "type", content = "data")]
 pub enum AppResponse {
-    /// This request/response is unimplemented
+    /// This request is unimplemented
     Unimplemented(AppRequest),
 
     /// Can occur in response to any [`AppRequest`].
     ///
     /// There has been an error during the handling of the request.
-    /// See [`ExternalApiWireError`] for variants.
-    ///
-    /// [`AppRequest`]: enum.AppRequest.html
-    /// [`ExternalApiWireError`]: error/enum.ExternalApiWireError.html
     Error(ExternalApiWireError),
 
     /// The succesful response to an [`AppRequest::AppInfo`].
     ///
-    /// Option will be `None` if there is no installed app with the given `installed_app_id` value from the request.
-    /// Check out [`InstalledApp`] for details on when the Option is `Some<InstalledAppInfo>`
-    ///
-    /// [`InstalledApp`]: ../../../holochain_types/app/struct.InstalledApp.html
-    /// [`AppRequest::AppInfo`]: enum.AppRequest.html#variant.AppInfo
+    /// Option will be `None` if there is no installed app with the given `installed_app_id`.
+    /// Check out [`InstalledApp`] for details on when the option is `Some<InstalledAppInfo>`
     AppInfo(Option<InstalledAppInfo>),
 
     /// The successful response to an [`AppRequest::ZomeCall`].
     ///
-    /// Note that [`ExternIO`] is simply a structure of [`SerializedBytes`] so the client will have
-    /// to decode this response back into the data provided by the Zome using a [msgpack](https://msgpack.org/) library to utilize it.
+    /// Note that [`ExternIO`] is simply a structure of [`struct@SerializedBytes`], so the client will have
+    /// to decode this response back into the data provided by the zome using a [msgpack] library to utilize it.
     ///
-    /// [`AppRequest::ZomeCall`]: enum.AppRequest.html#variant.ZomeCall
-    /// [`ExternIO`]: ../../../holochain_zome_types/zome_io/struct.ExternIO.html
-    /// [`SerializedBytes`]: ../../../holochain_zome_types/query/struct.SerializedBytes.html
+    /// [msgpack]: https://msgpack.org/
     ZomeCall(Box<ExternIO>),
 
-    /// DEPRECATED. See `ZomeCall`.
+    #[deprecated = "use ZomeCall"]
```

Comment:
Does this cause a warning? If so it may make some things fail to compile because we deny warnings

Reply:
It doesn't cause a warning. I'm always building the docs locally first and there weren't any warnings. I removed this because it's displayed double in the docs
<img width="408" alt="image" src="https://user-images.githubusercontent.com/28270981/169529099-52f8bd33-c613-481f-a8db-fbda1407f7e7.png">